### PR TITLE
Moved navbar, added styling for mobiles

### PR DESCRIPTION
**Pull request by gingerbreadpup**
Moved the navbar to the top so we don't have the keyboard issues, Charlie. For the bottom nav to have been effective I also would have wanted the bottom nav to disappear on scroll down, and reappear on scroll up - maybe further down the line we can look into it?

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ var webpack = require('webpack')
 var HtmlWebpackPlugin = require('html-webpack-plugin')
 
 if (process.env.NODE_ENV === 'production') {
-    var publicPath = 'https://neontribe.github.io/ARCVMarket/';
```
Comment (charlesstrange2) [inline webpack.config.js]: While you can dev locally with ./ it needs to be the full https to run if pushed to github pages (at the moment - needs fixing); avoid pushing webpack.config.js OR put it back before you do :-)

**Review by charlesstrange2 — CHANGES_REQUESTED** [review]
Code's good, just wtach webpack.config.js changes

File: webpack.config.js (hunk 1 above)
Comment (gingerbreadpup) [reply webpack.config.js]: I don't even recall making this change (maybe it was when we were trying to get stuff to work the other day?) . Will put it back in.

**Comment by charlesstrange2** [comment]
I think it was when we were getting it working for you - It's to do with
the build rather than dev side.

**Review by charlesstrange2 — APPROVED** [review]
:shipit:

**Pull request merged**